### DISCUSSION TITLE
심사 거절 되었는데 왜 그런지 잘 모르겠습니다.

### DISCUSSION THREAD

**Discussion by jhYoon90** [naverz/zepeto-studio-kor#694] [5. Q&A]
![refusal](https://user-images.githubusercontent.com/99247489/166393809-c1106fa0-ce83-479b-8d91-64ea36ed02ca.PNG)

로그인한 캐릭터로 캐릭터 생성되도록 코드도 수정하여 제출했는데도 왜 저걸 되었는지 모르겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 캐릭터 생성 코드에 zepeto id기준으로 작성하면 다른 사람이 플레이해도 개발자님의 id가 보여집니다.
아래 가이드에 나온 예제 코드를 참고하셔서 싱글 월드/멀티 월드에 맞춰서 코드 수정하시면 해결될 것입니다 :)
https://studio.zepeto.me/guides/zepetoplayer

**Reply by jhYoon90** [reply]
감사합니다
